backend/src/test/java/com/benine/backend/http/CreatePresetHandlerTest.java

```diff
@@ -70,7 +79,6 @@ public void testCameraID() throws Exception{
 
     getHandler().handle(target, requestMock, httprequestMock, httpresponseMock);
 
-    verify(presetController).addPreset(preset);
```

Comment:
I don't think it is a good idea to delete a test statement to make the test pass.